components/board/pi/impl/board.go

```diff
@@ -482,21 +499,25 @@ func (pi *piPigpio) DigitalInterruptNames() []string {
 	return names
 }
 
+// AnalogReaderByName returns an analog reader by name.
 func (pi *piPigpio) AnalogReaderByName(name string) (board.AnalogReader, bool) {
 	a, ok := pi.analogs[name]
 	return a, ok
 }
 
+// SPIByName returns an SPI bus by name.
 func (pi *piPigpio) SPIByName(name string) (board.SPI, bool) {
 	s, ok := pi.spis[name]
 	return s, ok
 }
 
+// I2CByName returns an I2C by name.
 func (pi *piPigpio) I2CByName(name string) (board.I2C, bool) {
 	s, ok := pi.i2cs[name]
 	return s, ok
 }
 
+// DigitalInterruptByName returns a digital interrupt by name.
```

Comment:
[minor] worth explaining here that if a digital interrupt has not been created for a pin on initial board setup, then this will attempt to create one on the fly and register it with the board, interpreting the name as a pin number

Reply:
Good point.